packages/nocodb/src/modules/jobs/jobs/at-import/at-import.processor.ts

```diff
@@ -328,13 +328,31 @@ export class AtImportProcessor {
         (x) => x.column_name?.toLowerCase() === col_name?.toLowerCase(),
       );
       if (duplicateTitle) {
-        if (enableErrorLogs) console.log(`## Duplicate title ${col_alias}`);
+        let iterator = 1;
+        while (
+          table.columns.find(
+            (x) =>
+              x.title?.toLowerCase() ===
+              `${col_alias}_${++iterator}`?.toLowerCase(),
+          )
+        ) {}
+        col_alias = `${col_alias}_${iterator}`;
+      }
+      if (duplicateColumn) {
+        let iterator = 1;
+        while (
+          table.columns.find(
+            (x) =>
+              x.column_name?.toLowerCase() ===
+              `${col_name}_${++iterator}`?.toLowerCase(),
+          )
+        ) {}
```

Comment:
increasing iterator here would be more easy to understand.

Reply:
I've revised them for better readability